### PR TITLE
notary blog post

### PR DESCRIPTION
**Pull request by hrbrmstr**
## Description

I made a blog post! :-) 

I think I got the YAML right and the three images are sourced from the notary repo.

### CODE REVIEW

File: _drafts/2017-07-25-notary.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+GitHub (and other public code repositories) are wonderful places where folks can collaborate and share creations. They are also fraught with peril. This is easily demonstrated by this proof-of-concept R package [`rpwnd`](https://github.com/hrbrmstr/rpwnd). Since GitHub is the most popular public R package development area, we'll focus on it for the remainder of this section.
+
+One way to begin to migitgate the threat of GitHub package distribution is to impose some rules and provide an means to ensure some level of authenticity at the author and release level. To that end, we have two core functions: `install_release()` and `validate_release()` that rely on a setting that most of you likely do not have enabled in GitHub - [PGP keys](https://github.com/settings/keys). You can read up on [GitHub & PGP](https://help.github.com/articles/signing-commits-with-gpg/) but you should really keep one finger on this page (again) and go check out [_A Git Horror Story: Repository Integrity With Signed Commits_](https://mikegerwitz.com/papers/git-horror-story.html).
```
Comment (stefaniebutland) [inline _drafts/2017-07-25-notary.md]: s/migitgate/mitigate/

Comment (stefaniebutland) [inline _drafts/2017-07-25-notary.md]: s/provide an means/provide a means/

File: _drafts/2017-07-25-notary.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+By having a similar set of signing and validation idioms, it will be possible to ensure that what you think you're getting from a CRAN repository is what was approved by the CRAN team. We still need to get one "secure" mirror setup to enable a proof-of-concept, so stay tuned for advancements in this area.
+
+## A Sheriff for the Wild, Wild West (i.e. Making GitHub Safer)
+
+<img src="https://raw.githubusercontent.com/ropenscilabs/notary/master/img/badge.png" align="right"/> While the CRAN distribution model is not perfect, it's Fort Knox compared to GitHub. 
+
+Oh, but before we go into that, you should check out some extremely cutting edge functionality Hadley and others are putting into `purrr`. Just do a quick `devtools::install_github("hadlley/purrr")` bring up the help for the new *threaded* parallel execution of `map()`: `map_t()`.
```
Comment (stefaniebutland) [inline _drafts/2017-07-25-notary.md]: I suggest linking people to their GitHub or twitter to help people find them
e.g. https://github.com/hadley

File: _drafts/2017-07-25-notary.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+Most of us who work in R just want to Get Stuff Done&trade;. We want a minimum amount of friction between ourselves and the data we need to wrangle, analyze, and visualize. We're focused on solving a problem or gaining insights into a new area of research. We rely on a rich, community-driven ecosystem of packages to help get our work done and likely make an unconscious assumption that there is a safety net out there, protecting us from harm.
+
+Unfortunately, I get to be "that guy" who comes along and shatters such assumptions. It's time to put our hard hats on, get our clipboards out, and take a safety inspection tour of R. Along the way, we'll introduce features and design concepts of our rOpenSci \#unconf17 project &mdash; the [`notary` package](https://github.com/ropenscilabs/notary) &mdash; that are aimed at making working in R a bit safer and more secure.
+
+## Meet The Team
+
+Since we say "we" quite a bit in this post, here are the folks that are represented by those two letters:
+
+- Rich FitzJohn
```
Comment (stefaniebutland) [inline _drafts/2017-07-25-notary.md]: - [Rich FitzJohn](https://github.com/richfitz)
- [Oliver Keyes](https://github.com/ironholds)
- [Stephanie Locke](https://github.com/stephlocke)
- [Jeroen Ooms](https://github.com/jeroen)
- [Bob Rudis](https://github.com/hrbrmstr/)

File: _drafts/2017-07-25-notary.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+To that end, the team came up with the concept of _signing_ packages (hence the `notary` name). Without taking you down a deep dive into [digitial signatures](https://en.wikipedia.org/wiki/Digital_signature), you're already familiar with this concept if you use something like an iOS-based device (i.e. iPhone or iPad) and have downloaded an app from the Apple app store. A developer applies for a developer account with Apple. They get a key. They make an app. They digitally sign the app with the key they received. Apple reviews the app and (usually)eventually approves it. The signed app goes into the store and your iOS device (if you haven't "rooted" it) is configured to only run signed and approved apps.
```
Comment (stefaniebutland) [inline _drafts/2017-07-25-notary.md]: s/digitial/digital/

File: _drafts/2017-07-25-notary.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
+---
+name: notary
+layout: post_discourse
+title: notary: Signing & Verification of R Packages
```
Comment (stefaniebutland) [inline _drafts/2017-07-25-notary.md]: @hrbrmstr Ok if we make these edits then post?

**Comment by hrbrmstr** [comment]
absolutely.

On Tue, Jul 25, 2017 at 12:16 PM, Stefanie Butland <notifications@github.com

**Comment by sckott** [comment]
up ⬆️

**Pull request merged**